sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+    "            }\n",
+    "        },\n",
+    "        StoppingCondition={\n",
+    "            \"MaxRuntimeInSeconds\": 3600,\n",
+    "        },\n",
+    "        RoleArn=role,\n",
+    "    )\n",
+    "\n",
+    "    return processing_job_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is a brief explanation of inputs used above, for detailed documentation check [CreateProcessingJob API reference](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateProcessingJob.html):\n",
+    "* `AppSpecification`: Here we provide the region specific clarify image uri we fethced earlier\n",
+    "* `ProcessingInputs`: Clarify job requires that you provde two ProcessingInput parameters.\n",
+    "  * `InputName: analysis_config`: The analysis configuration JSON file for a SageMaker Clarify job must be specified as an Amazon S3 object with the InputName \"analysis_config\". We will be providing the example analysis_configs that we have provided with this notebook. \n",
+    "  * `InputName: dataset`, dataset fetched earlier provided here as an Amazon S3 object.\n",
+    "* `ProcessingOutputConfig`: The job also requires an output parameter, the output location as an Amazon S3 prefix with the OutputName \"analysis_result\". The S3UploadMode should be set to \"EndOfJob\", because the analysis results is generated at the end of the job. We will be providing here the `analysis_result_path` that we configured earlier.\n",
+    "* `ProcessingResources` contains the ClusterConfig specifying the ML compute instance_type we want to use and the count. SageMaker SHAP analysis is CPU-intensive, to speed up the analysis, use a better instance type, or add more instances to enable Spark parallelization.. The SageMaker Clarify job doesn’t use GPU.\n",
+    "* `StoppingCondition`: Using a maximum limit of 60 min for example job run. You can set the MaxRuntimeInSeconds of a SageMaker Clarify job to up to 7 days (604800 seconds). If the job cannot be completed within this time limit, it will be force-stopped and no analysis results are provided."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wait for processing job to complete\n",
+    "def wait_for_job(job_name):\n",
+    "    while sagemaker_client.describe_processing_job(ProcessingJobName=job_name)[\"ProcessingJobStatus\"] == \"InProgress\":\n",
+    "        print(\".\", end=\"\")\n",
+    "        time.sleep(60)\n",
+    "    print()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "........\n"
+     ]
+    }
+   ],
+   "source": [
+    "processing_job_name = create_processing_job(bias_analysis_config_path, bias_analysis_result_path)\n",
+    "wait_for_job(processing_job_name)"
```

Comment:
Good to warn the user how long the job runs. E.g. "The job takes about 10 minutes to run."